src/Arcus.Observability.Telemetry.Core/Logging/RequestSourceSystem.cs

```diff
@@ -18,6 +18,11 @@ public enum RequestSourceSystem
         /// <summary>
         /// Specifies that the request-source is an Azure EventHubs.
         /// </summary>
-        AzureEventHubs = 4
+        AzureEventHubs = 4,
+
+        /// <summary>
+        /// Specifies that the request-source is a custom system.
+        /// </summary>
+        Custom = 8
```

Comment:
Is that backing integer used somewhere ?  Should we use a higher number ?

Reply:
It's mostly to be strict in using switches. If we add more types, unexpected scenario's could happen when interacting with the enum in a different way. It's basically a failsafe to make sure we know what value is behind each value.

Reply:
Ok, that's why I was asking if we should use a higher number for 'custom'.  (Or is it not necessary to have the highest backing intger assigned to 'custom').

Reply:
But, the enum is not a Flag - enum, so I don't see how you could apply bitwise operations to them.

Reply:
You could still use bitwise operations, it's just that if the number overlaps on a future number, we could have unexpected results.

Reply:
> Ok, that's why I was asking if we should use a higher number for 'custom'. (Or is it not necessary to have the highest backing intger assigned to 'custom').

We could provide a higher number, that's true, to maybe make sure that it's cleaner, but on the other hand, we don't use these integers. Only as a failsafe.